velox/dwio/text/writer/TextWriter.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/dwio/text/writer/TextWriter.h"
+
+#include <utility>
+#include "velox/common/base/Pointers.h"
+#include "velox/exec/MemoryReclaimer.h"
+
+namespace facebook::velox::text {
+
+static std::string encodeColumnCellValue(
+    const std::shared_ptr<DecodedVector>& decodedColumnVector,
+    const TypePtr& type,
+    vector_size_t row) {
+  if (decodedColumnVector->isNullAt(row)) {
+    return TextWriter::nullData;
+  }
+
+  switch (type->kind()) {
+    case TypeKind::BOOLEAN:
+      return std::to_string(decodedColumnVector->valueAt<bool>(row));
```

Comment:
this might return 1 for true and 0 for false? Is this expected?

fyi: folly::to<bool> return "true" for a boolean true

Reply:
Yes, this is also how java stores it